### PR TITLE
W-16010428: aditional parameters configured for MMP deploy to CH2

### PR DESCRIPTION
**Pull request by aguozden**
As requested by CH2, 3 parameters were added for MMP deploys to CH2 :

- Boolean PersistentObjectStore
- Boolean tracingEnabled
- Class autoscaling { 
            Boolean enabled,
            Integer minReplicas,
            Integer maxReplicas
            }

Reference: [https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a[…]orm/amc-application-manager/minor/4.0/console/type/%23108/](https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/type/%23108/)

### CODE REVIEW

File: mule-deployer/src/main/java/org/mule/tools/model/anypoint/RuntimeFabricOnPremiseDeploymentSettings.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public RuntimeFabricOnPremiseDeploymentSettings() {
   public RuntimeFabricOnPremiseDeploymentSettings(RuntimeFabricOnPremiseDeploymentSettings settings) {
     super(settings);
     resources = settings.resources;
-    persistentObjectStore = settings.persistentObjectStore;
+    //persistentObjectStore = settings.persistentObjectStore;
```
Comment (egrave) [inline mule-deployer/src/main/java/org/mule/tools/model/anypoint/RuntimeFabricOnPremiseDeploymentSettings.java:36]: same here

File: mule-deployer/src/main/java/org/mule/tools/deployment/cloudhub2/RequestBuilderCh2.java [hunk 2]
```diff
@@ -88,19 +88,16 @@ protected RuntimeFabricDeploymentSettings resolveDeploymentSettings(RuntimeFabri
   }
 
   public Object createConfiguration() {
-    Map<String, Object> properties = null;
```
Comment (egrave) [inline mule-deployer/src/main/java/org/mule/tools/deployment/cloudhub2/RequestBuilderCh2.java:91]: it seems that here you're undoing your latest change https://github.com/mulesoft/mule-maven-plugin/commit/3d4fc0b3f53fa5f686238bb9944597de567d3224

File: mule-deployer/src/main/java/org/mule/tools/model/anypoint/RuntimeFabricOnPremiseDeploymentSettings.java [hunk 3]
```diff
@@ -22,8 +22,8 @@ public class RuntimeFabricOnPremiseDeploymentSettings extends RuntimeFabricDeplo
   @Parameter
   protected Jvm jvm;
 
-  @Parameter
-  protected boolean persistentObjectStore;
+  //@Parameter
```
[the remark this answers is not in the document]
Comment (aguozden) [reply mule-deployer/src/main/java/org/mule/tools/model/anypoint/RuntimeFabricOnPremiseDeploymentSettings.java]: Thanks, done !

File: mule-deployer/src/main/java/org/mule/tools/model/anypoint/RuntimeFabricOnPremiseDeploymentSettings.java (hunk 1 above)
Comment (aguozden) [reply mule-deployer/src/main/java/org/mule/tools/model/anypoint/RuntimeFabricOnPremiseDeploymentSettings.java]: Thanks, done !

File: mule-deployer/src/main/java/org/mule/tools/deployment/cloudhub2/RequestBuilderCh2.java (hunk 2 above)
Comment (aguozden) [reply mule-deployer/src/main/java/org/mule/tools/deployment/cloudhub2/RequestBuilderCh2.java]: thanks.. updated and tested

**Review by egrave — APPROVED** [review]

**Comment by egrave** [comment]
!gate waiver add Sonar

**Pull request merged**